tasks/update_go.py

```diff
@@ -10,6 +10,24 @@
 
 GO_VERSION_FILE = "./.go-version"
 
+# list of references of Go versions
+# each tuple is (path, pre_pattern, post_pattern, minor), where
+# - path is the path of the file to update
+# - pre_pattern and post_pattern delimit the version to update
+# - minor is whether the version in the match is a minor version of a bugfix version
```

Comment:
When using bool values, I prefer to avoid double negatives in order to avoid confusion during usage. Usually that means the `true` case should represent something "more", and `false` should present something "less". In this case, `minor` is true when the version is shorter like "1.20", but false when the version is longer like "1.20.8". I would suggest:

1) Change this name to `is_full` and describe it as true when the full version appears rather than the major+minor.
2) Invert the True / False values in this list.
3) Swap the conditionals that are currently using `minor` in `_update_references`, `_get_pattern`, etc.

Reply:
That makes sense ! I'll update it

Reply:
I actually renamed it `is_bugfix` as I think it's more explicit that `is_full`